dev/conductor/core/test/codesign_integration_test.dart

```diff
@@ -19,13 +21,36 @@ import './common.dart';
 
 /// Verify all binaries in the Flutter cache are expected by Conductor.
 void main() {
+  const Platform platform = LocalPlatform();
+  const FileSystem fileSystem = LocalFileSystem();
+  const ProcessManager processManager = LocalProcessManager();
+
+  late Directory tempDir;
+
+  /// Whether or not the test is running in the context of a CI release build.
+  ///
+  /// The LUCI_BRANCH env variable is set in the LUCI recipes repo_util module.
+  /// See
+  /// https://flutter.googlesource.com/recipes/+/refs/heads/main/recipe_modules/repo_util/api.py.
```

Comment:
Relying on LUCI_BRANCH SGTM. I would just add that we should include a doc in the recipes link saying it has a dependency here.

Reply:
Sounds good.

I was also thinking that I should update the recipes and make sure this var is set post-submit too.